tutorials/hosted_phoenix/hosted_phoenix_openai_tutorial.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    px_client.get_spans_dataframe(project_name="rechat", root_spans_only=True)
</pre><p>I don't see project name in any of the code above. You sure this works? also why the root_spans_only flag here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>